addons/purchase/migrations/12.0.1.2/post-migration.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2019 Eficent <http://www.eficent.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+def fill_purchase_order_user_id(cr):
+    cr.execute(
+        """
+        UPDATE purchase_order
+        SET user_id = create_uid
+        WHERE user_id IS NULL
+        """
+    )
+
+
+def reset_domain_purchase_rfq(cr):
+    cr.execute(
+        """
+        UPDATE ir_act_window iaw
+        SET domain = NULL
+        FROM ir_model_data imd
+        WHERE AND imd.name = 'purchase_rfq' AND imd.res_id = iaw.id
```

Comment:
bad syntax: `WHERE AND`
I guess you meant `WHERE imd.name = 'purchase_rfq' AND imd.res_id = iaw.id` ?

Reply:
yes, thanks, it was a typo